draft-ietf-httpbis-semantics-latest.xml

```diff
@@ -1605,10 +1607,19 @@ Content-Type: text/plain
    as quoted-string (<xref target="quoted.strings"/>).
 </t>
 <t>
-   Because commas (",") are used as a generic delimiter between members of a
-   list-based field value, they need to be treated with care if they are
-   allowed as data within those members. Typically, list members that might contain a
-   comma are enclosed in a quoted-string.
+   A field's value often contains one or more members, separated by commas (",").
+   A field that allows multiple members is referred to as a "list-based" field.
+   A field that only allows a single member is referred to as a "singleton" field.
+</t>
```

Comment:
Maybe change the first sentence to:

"Commas (",") often are used to separate members in field values."

?

Reply:
I don't think that effectively introduces the concept of members of a list.

Reply:
Ah, I didn't realize that this was supposed to introduce a term/concept. That makes it even more important to get it right. Consider:

    If-Match: "a, b"

or

    Location: foo,bar

In this case, do we really call '"a' and 'b"' (or 'foo' and 'bar') "members of a list"?

Reply:
I see your point; will revise.